### PR TITLE
#DP-1547 feat: added add from library button  enable/disable based on config

### PR DESCRIPTION
**Pull request by santoshilimi**
… config

### CODE REVIEW

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts [hunk 1]
```diff
@@ -436,6 +439,14 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
         break;
     }
   }
+  addFromLibraryButton(nodeLevel) {
+    if (nodeLevel === 0) {
+     _.isEmpty(_.get(this.config, 'children')) ? this.showLibraryButton = false : this.showLibraryButton = true;
```
Comment (pallakartheekreddy) [inline projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts:444]: use below code 
```
this.showLibraryButton = _.isEmpty(_.get(this.config, 'children')) ? false : true;
```

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts [hunk 2]
```diff
@@ -436,6 +439,14 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
...
+    } else {
+     const hierarchylevelData =  this.config.hierarchy[`level${nodeLevel}`];
+     _.isEmpty(_.get(hierarchylevelData, 'children')) ? this.showLibraryButton = false : this.showLibraryButton = true;
```
Comment (pallakartheekreddy) [inline projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts:447]: same here

**Review by pallakartheekreddy — CHANGES_REQUESTED** [review]

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts (hunk 1 above)
Comment (santoshilimi) [reply projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts]: Done

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts (hunk 2 above)
Comment (santoshilimi) [reply projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts]: Done

**Pull request merged**